sys-auth/pam_u2f/pam_u2f-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
This package doesn't get frequent updates. So this might be a good time to bump its EAPI with this opportunity. Required changes are minimal: move the dependency on `virtual/pkgconfig` from `DEPEND` to `BDEPEND` and have `DEPEND` and `RDEPEND` be identical.